caddy/frankenphp/Caddyfile

```diff
@@ -1,37 +1,18 @@
 {
 	# Debug
-    {$CADDY_DEBUG}
-
-	frankenphp {
-		#worker /path/to/your/worker.php
+	{$CADDY_DEBUG}
+    frankenphp index {
```

Comment:
indentation to fix.

Shouldn't be this directive outside of the global block (as `php_fastcgi`)?

Also, why do we need to pass the `index` matcher? It doesn't look defined.

Also, by default just having `frankenphp` should be enough as for `php_fastcgi`.